frontend/src/components/page-about/index/belief.js

```diff
@@ -1,104 +1,67 @@
 import * as React from "react"
-import arrowImage from "../../../images/chevron-right.png"
+import { Collapsible } from "../../collapsible"
 import {
     container,
     titleContainer,
     middleContainer,
-    arrow,
-    subHead,
-    box,
 } from "../../../css/belief.module.css"
 
+const BibleProp = "We believe the Bible, both the Old and New Testaments, is the only inspired, infallible, and authoritative Word of God. It is the supreme source of truth for Christian faith and living.";
+const GodProp = "We believe in one God, Creator of all things, infinitely perfect and eternally existing in three persons: Father, Son, and Holy Spirit.";
+const JesusProp = "We believe that Jesus Christ is true (fully) God and true (fully) man, having been conceived of the Holy Spirit and born of the virgin Mary. He died on the cross, the complete and final sacrifice for our sins according to the Scriptures. He arose bodily from the dead, and ascended into heaven where, at God’s right hand, He intercedes for His people and rules as Lord over all.";
+const HSProp = "We believe that the ministry of the Holy Spirit is to glorify the Lord Jesus Christ, and during this age to convict the world of sin. He also regenerates the believing sinner, indwelling, guiding, instructing, and empowering us for godly living and sacrifice.";
```

Comment:
Nit, optional: For these, I think *Section or *SectionText would be more descriptive than *Prop. Also, in general it's okay to have longer variable names for the sake of descriptiveness. `HolySpiritSectionText` makes it more clear what this variable is than `HSProp`